include/fmt/ostream.h

```diff
@@ -129,14 +129,22 @@ template <typename T> struct streamed_view { const T& value; };
 
 // Formats an object of type T that has an overloaded ostream operator<<.
 template <typename Char>
-struct basic_ostream_formatter : formatter<basic_string_view<Char>, Char> {
+struct basic_ostream_formatter {
+private:
+  formatter<basic_string_view<Char>, Char> underlying_;
+
```

Comment:
Why is this change needed?
Inheritance from formatter is allowed in the documentation (https://fmt.dev/latest/api.html):

> You can also reuse existing formatters via inheritance or composition, for example:
> ...
> ``template <> struct fmt::formatter<color>: formatter<string_view> {``
> ...
> Since parse is inherited from formatter<string_view> it will recognize all string format specifications, for example

Reply:
To avoid inheriting `set_debug_format()`